checkout.py

```diff
@@ -4,10 +4,13 @@
 
 @author: vjham
 """
+from datetime import datetime
 from validation import is_clerk_input_valid
 
 items = []
```

Comment:
So, I think a lot of times the convention is to keep all the constants together at the top of the file. So we might want to re-arrange some code here. Maybe to look more like this:

```
GROCERY_STORE_NAME = "Shop N Bag"
DATE_TIME_FORMAT = date_and_time.strftime("%B %d %Y %H:%M:%S") # Example: August 29 2021 13:14:55


items = []
date_and_time = datetime.now()
```

Just to help keep the file organized

Reply:
This can not happen because we can not use a variable before assigning it but we can do it this way 
GROCERY_STORE_NAME = "Shop N Bag"
date_and_time = datetime.now()
DATE_TIME_FORMAT = date_and_time.strftime("%B %d %Y %H:%M:%S") # Example: August 29 2021 13:14:55
items = []

Reply:
oh!!! that's my bad for not reading the code thoroughly. I totally totally misread!!

In that case! 
I would save the format as a constant like this:

`DATE_TIME_FORMAT = "%B %d %Y %H:%M:%S" # Example: August 29 2021 13:14:55`

and then in `our create_custom_receipt()` method, instead of just using `DATE_TIME_FORMAT`, I would do 
`date_and_time.strftime(DATE_TIME_FORMAT)`

or you could create a variable (in the create_custom_receipt() method) like this:
`formatted_date = date_and_time.strftime(DATE_TIME_FORMAT)`
and then use `formatted_date` in the return string.

I thought that DATE_TIME_FORMAT was just the format, I didn't realize it was the actual formatted date and time. That's on me for not paying closer attention.